dgl/Window.hpp

```diff
@@ -119,6 +119,10 @@ class Window
     Application& getApp() const noexcept;
     intptr_t getWindowId() const noexcept;
 
+#if defined(HAVE_DCAIRO)
+    cairo_t* getGraphics() const noexcept;
```

Comment:
Can we call this getContext instead of getGraphics? Does it make sense?

Reply:
I have no particular preference. I renamed it.